vioscsi/trace.h

```diff
@@ -83,13 +110,21 @@ extern int nVioscsiDebugLevel;
 #define VioScsiDbgBreak()
 #endif
 
```

Comment:
I'm thinking this would be better:
```diff
-#if DBG
 #define RhelDbgPrint(Level, MSG, ...) \
     if ((!bDebugPrint) || Level > nVioscsiDebugLevel) {} \
     else VirtioDebugPrintProc (MSG, __VA_ARGS__)
 #define VioScsiDbgBreak()\
     if (KD_DEBUGGER_ENABLED && !KD_DEBUGGER_NOT_PRESENT) DbgBreakPoint();
-#else
-#define RhelDbgPrint(Level, MSG, ...)
-#define VioScsiDbgBreak()
-#endif
```

Reply:
@vrozenfe and @YanVugenfirer,
Just to clarify the NOT DBG path will never get hit here. I don't believe it should as producing kdebug and ETW output simultaneously would defeat the performance gains from running ETW only. If for some reason we do want to do this, then we should probably use the `#if !defined(DBG)` syntax rather than `#if DBG` and relocate this part outside the `#if !defined(EVENT_TRACING)` conditional, or perform some other refactoring so the relevant branches are executed where needed.